features/locale/switching_locales_as_admin.feature

```diff
@@ -0,0 +1,57 @@
+@locales
+Feature: Switching locales on admin's panel
+    In order to see the panel in my preferred language
+    As an Administrator
+    I want to be able to switch locales
+
+    Background:
+        Given the store operates on a single channel
+        And that channel allows to shop using "English (United States)", "Spanish (Mexico)" and "French (Canada)" locales
+        But the locale "French (Canada)" is disabled
+        And it uses the "English (United States)" locale by default
+        And I am logged in as an administrator
+
+    @ui @todo
+    Scenario: Viewing the administrator's panel in the default locale
+        When I open administration dashboard
+        Then I should be viewing it in "English (United States)" locale
+
+    @ui @todo
+    Scenario: Changing the panel's locale
+        When I open administration dashboard
+        And I switch to the "Spanish (Mexico)" locale
+        Then I should be viewing the panel in "Spanish (Mexico)"
+
+    @ui @todo
+    Scenario: Admin's panel can be viewed in a disabled locale
+        When I open administration dashboard
+        And I switch to the "French (Canada)" locale
+        Then I should be viewing the panel in "French (Canada)"
+
+    @ui @todo
+    Scenario: Changing panel's locale doesn't change shop's locale
+        Given I am using "Spanish (Mexico)" locale for my panel
+        When I visit this channel's homepage
```

Comment:
This refers to the channel in background, but it is not very clear when the channel code/name is not mentioned in the scenario itself.